### DISCUSSION TITLE
Error after installation - yarn add react-horizontal-scrolling-menu

### DISCUSSION THREAD

**Discussion by mircobarachetti** [asmyshlyaev177/react-horizontal-scrolling-menu#210] [General]
Hello, I've a problem with this solution because my web app doesn't start after the command "yarn add react-horizontal-scrolling-menu" (version 3.1.1).

I haven't this problem if I install the version 2.0.1. After this version I've alwasy this issue.

Could you help me to fix?

In attachment my package.txt and the error.
[package.txt](https://github.com/asmyshlyaev177/react-horizontal-scrolling-menu/files/9259651/package.txt)
![error](https://user-images.githubusercontent.com/9607467/182848296-5e2d9fc6-3014-4b2b-a847-074f1859b46b.JPG)

**Comment by asmyshlyaev177** [comment]
Can you update your typescript version?

**Reply by mircobarachetti** [reply]
Thanks!! Now it's working!
